Lib/distutils/util.py

```diff
@@ -295,8 +295,8 @@ def strtobool (val):
     """Convert a string representation of truth to true (1) or false (0).
 
     True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
-    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
-    'val' is anything else.
+    are 'n', 'no', 'f', 'false', 'off', and '0'. Alphabets are treated
+    case-insensitively. Raises ValueError if 'val' is anything else.
```

Comment:
LGTM. It also has a test so I guess it's worth documenting this. Maybe once accepted the corresponding docs page could also be updated ? 

Docs : https://github.com/python/cpython/blob/master/Doc/distutils/apiref.rst 
Test : https://github.com/python/cpython/blob/79da388a4016e24c4258dcc62cd0fa9dde0acb5b/Lib/distutils/tests/test_util.py#L271

Reply:
Good point. Doing it now.

Reply:
@tirkarthi See commit f2fe330 .